### DISCUSSION TITLE
How does create-release.ps1 work?

### DISCUSSION THREAD

**Discussion by rdvansloten** [actions/virtual-environments#5809] [General questions] [answered]
I've created a vhd using the ubuntu2004.yml successfully and uploaded it to my Storage Account, but now the pipeline is failing on the create release (create-release.ps1) part. It's expecting a release pipeline? I have no idea what this thing wants from me or what I should do. The step is also not optional, so the run fails (even though my image is created)

Anyone help? I'll update the docs if I figure out what this thing wants.

**Comment by al-cheb** [comment, marked as answer]
The `create-release.ps1` script is for internal use only and you don't have to use it in CI. That's why we don't have docs about it.

**Reply by rdvansloten** [reply]
Any chance you'll release the Release Pipeline this is attached to? I assume it's used to upload the VM image to the Image Gallery?

**Comment by al-cheb** [comment]
Any chance you'll release the Release Pipeline this is attached to? - We don't have any plans in the near future.
I assume it's used to upload the VM image to the Image Gallery? - It creates a release into Azure DevOps organization to deploy a single test vm based on the generated template in Azure.
